.github/workflows/build.yml

```diff
@@ -38,14 +42,14 @@ jobs:
       # (e.g. - apt-get: gettext, etc; pip: circuitpython-build-tools, requirements.txt; etc.)
       run: |
         source actions-ci/install.sh
-    - name: Pip install pylint, black, & Sphinx
+    - name: Pip install pylint, Sphinx, pre-commit
```

Comment:
Has there already been a discussion about switching from black to pre-commit? I'm not opposed to it, and after looking into it, it seems like a good option, but I think we may want to discuss it before changing it.

Reply:
Also, from what I can tell, pre-commit is setup to actually modify the files. We had decided that we shouldn't do that since black and pylint do disagree every so often.

Reply:
This was discussed briefly as part of https://github.com/adafruit/circuitpython/issues/2887

This configuration is taken directly from https://github.com/adafruit/Adafruit_CircuitPython_TestRepo/commit/18e383f76140cc102cbd4bd6c53cc9f132961fbf

And is already applied to ESP32SPI as well (https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/commit/2077d83a49a0e614fae232f8792dcf6c980909ea).

Reply:
Great. Thanks for linking that